scripts/deploy_assisted_service.sh

```diff
@@ -8,6 +8,7 @@ export SERVICE_NAME=assisted-service
 export NAMESPACE=${NAMESPACE:-assisted-installer}
 export SERVICE_URL=$(get_main_ip)
 export SERVICE_PORT=${SERVICE_PORT:-6000}
+export SERVICE_BASE_URL="http://${SERVICE_URL}:${SERVICE_PORT}"
```

Comment:
Why http?

Reply:
cause it is still http

Reply:
when we will not support it lets move to https @empovit ?

Reply:
Some environments can't have HTTPS. SSL termination at ingress doesn't work in minikube where port forwarding is involved, we already discussed it. I will add a flag when HTTPS support is ready.